web/yo/app/scripts/controllers/nav.js

```diff
@@ -23,12 +23,13 @@ angular.module('oncokbApp')
             filterTabs.push('genes');
```

Comment:
Could we just write like this?
```
var filterTabs = ['genes', 'queues', 'therapies', 'tools'];  
```

Reply:
This is previous setting. I think it's ok.